File: 1039-find-the-town-judge/find-the-town-judge.py
class Solution:
    def findJudge(self, n: int, trust: List[List[int]]) -> int:
        trust_counts = [0] * (n + 1)
        for a, b in trust:
            trust_counts[a] -= 1
            trust_counts[b] += 1
        for i in range(1, n + 1):
            if trust_counts[i] == n - 1:
                return i
        return -1

File: 1039-find-the-town-judge/find-the-town-judge.js
var findJudge = function(n, trust) {
     var trustCounts = new Array(n + 1).fill(0);
        for(var i = 0; i < trust.length; i++) {
            trustCounts[trust[i][0]]--;
            trustCounts[trust[i][1]]++;
        }
        for(var i = 1; i <= n; i++) {
            if(trustCounts[i] === n - 1) {
                return i;
            }
        }
        return -1;
};